server/server.lua

```diff
@@ -6,5 +6,21 @@ end)
 
 RegisterNetEvent("qbx-customs:server:buyCart", function(cart)
     local src = source
+    
+    local plate = cart.plate
+    local mods = cart.mods
 
-end)
\ No newline at end of file
+    if not IsVehicleOwned(plate) then return end
+    exports.oxmysql:execute('UPDATE player_vehicles SET mods = ? WHERE plate = ?', {json.encode(mods), plate})
+    
+end)
+
+
+
+local function IsVehicleOwned(plate)
+    local result = exports.oxmysql:scalarSync('SELECT plate FROM player_vehicles WHERE plate = ?', {plate})
+    if result then
+        return true
+    end
+    return false
+end
```

Comment:
Would prefer these database queries to be added to qb-core/server/storage.lua and then exported to be called here, but this is fine for now.